pimd/pim_str.h

```diff
@@ -37,6 +40,26 @@
 #define pim_inet4_dump prefix_mcast_inet4_dump
 #define pim_str_sg_set prefix_sg2str
 
+static inline void pim_addr_copy(pim_addr *dest, pim_addr *source)
+{
+	dest->s_addr = source->s_addr;
+}
+
+static inline int pim_is_addr_any(pim_addr addr)
+{
+	return (addr.s_addr == INADDR_ANY);
+}
+
+static inline int pim_addr_compare(pim_addr addr1, pim_addr addr2)
+{
+	return IPV4_ADDR_CMP(&addr1, &addr2);
+}
+
+static inline int pim_addr_is_same(pim_addr addr1, pim_addr addr2)
```

Comment:
- I'm not a huge fan of having both `cmp` and `same` functions. `same` is `!cmp`; C programmers should IMHO understand this from `memcmp` and `strcmp` - there's no `memsame` or `strsame` either.
- if we do add this, this should return ´bool` not `int`

Reply:
Fixed.